### ISSUE TITLE
CJS output has `const`

### ISSUE THREAD

**Issue by peter-mouland** [jaredLunde/render-props#2]
It looks like the output for CJS isn't ready for browsers that don't support modules.

The build tool/step needs to be updated to convert newer syntax like const into alternatives that older browsers support

**Comment by jaredLunde** [comment]
The CJS builds are only intended to support Node so people can require() it in their terminal and in tests. I’m not making decisions for the user about which browsers need to be supported. Some people would want IE11 support for example and I personally never need to support this browser in my work anymore. I also opted not to make a UMD build for the browser because most people are using Webpack, Rollup, Parcel, etc and I don’t want to maintain a Rollup config to support that small %. Finally, CRA transpiles node_modules because they know you can’t trust libraries not to make a mistake in their Babel config or reliably support a huge % of browsers - I’ve personally run into that with libraries using babel/preset-env that don’t cover Object.keys, Object.values.

tl;dr; IMO it’s the user’s responsibility to transpile for their own targets.
